### PR TITLE
Remove comments after pragma restore to reduce clutter

### PR DESCRIPTION
**Pull request by paul1956**
PR Feedback from VB applied to C#

## Proposed changes
- Remove comment after pragma restore

## Customer Impact
- None

## Regression? 
-  No

## Risk
- None - no code changes.

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/12128)

### CODE REVIEW

File: src/Common/src/Obsoletions.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ internal static class Obsoletions
 
 #pragma warning disable WFDEV003 // Type or member is obsolete
     internal const string DomainItemAccessibleObjectMessage = $"{nameof(DomainUpDown.DomainItemAccessibleObject)} is no longer used to provide accessible support for {nameof(DomainUpDown)} items.";
-#pragma warning restore WFDEV003 // Type or member is obsolete
```
Comment (Tanya-Solyanik) [inline src/Common/src/Obsoletions.cs:22]: I find these comments helpful because I don't remember what the warnig is about.

Comment (paul1956) [reply src/Common/src/Obsoletions.cs:22]: @Tanya-Solyanik Its 2 lines above and was PR feedback, maybe there should be a rule about how many lines separates disable from enable.

Its teams decision, I just need 1 answer.

If you decide to leave alone I can just close PR without Merge.

Comment (Tanya-Solyanik) [reply src/Common/src/Obsoletions.cs:22]: Makes sense.

**Review by Tanya-Solyanik — APPROVED** [review]
Thank you!

**Pull request merged**